config.default

```diff
@@ -1,18 +1,18 @@
-## This file contains the default values of all possible configuration options 
-## used to run analysis tasks.  Do not modify options in this file direct.  
+## This file contains the default values of all possible configuration options
+## used to run analysis tasks.  Do not modify options in this file direct.
```

Comment:
Here and elsewhere in this file, it is just automatically deleting trailing whitespace.  No other changes were made.